src/libratbag.c

```diff
@@ -1197,6 +1198,11 @@ ratbag_resolution_set_active(struct ratbag_resolution *resolution)
 	struct ratbag_profile *profile = resolution->profile;
 	struct ratbag_resolution *res;
 
+	if (resolution->is_disabled) {
+		log_error(profile->device->ratbag,"Setting the active resolution to a disabled resolution is not allowed\n");
```

Comment:
style: space: `, "`
nitpick: make the message lowercase and print the device name

```c
struct ratbag_device *device = profile->device;
log_error(device->ratbag, "%s: setting the active resolution to a disabled resolution is not allowed\n", device->name);
```